app/server/models/transfer_usage.py

```diff
@@ -29,3 +31,13 @@ def icon(self, icon):
         if icon not in MATERIAL_COMMUNITY_ICONS:
             raise IconNotSupportedException('Icon {} not supported or found')
         self._icon = icon
+
+    @classmethod
+    def create_without_duplicate(cls, raw_name, icon, priority, default):
```

Comment:
would making name a unique attribute + having the setter for name strip and upper not achieve the same thing?

Reply:
Good idea :)